src/jobs/rss-news.js

```diff
@@ -50,7 +49,9 @@ async function processSchool(school) {
 			allCheckedNews = allCheckedNews.concat(checkedNews);
 			dbFeed.status = 'success';
 		} catch (err) {
-			logger.error(`Could not handle feed ${dbFeed.url} (${dbFeed._id}) for school ${school._id}`, err);
+			const message = `Could not handle feed '${dbFeed.url}' (${dbFeed._id}) for school '${school._id}'`;
```

Comment:
The array errors is the same array like line 66, you put the error message twice in it or?